server/target/target.go

```diff
@@ -36,7 +43,18 @@ const (
 	coverageCommand = "coverage"
 
 	// The number of distinct commits returned in GetTargetHistoryResponse.
-	targetPageSize = 20
+	targetHistoryPageSize = 20
+
+	// The number of targets returned in the GetTarget response.
+	targetPageSize = 12
+
+	// When returning a paginated list of all targets in an invocation with
+	// files expanded, stop returning targets after this many files have been
+	// expanded. This is a "soft" limit because the file list is not truncated,
+	// only the target list. So for example if a target contains 101 files, all
+	// 101 files will be returned, but only that target will be returned in the
+	// target page.
+	targetFilesSoftLimit = 100
```

Comment:
Do we need this?

Reply:
I think so - some targets can have a ton of files and some can have barely any. We should ideally paginate by number of files directly but the GetTarget API doesn't make that very easy, currently. We can maybe add a separate GetFiles endpoint later